usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -0,0 +1,141 @@
+DROP VIEW IF EXISTS award_delta_view;
+
+CREATE VIEW award_delta_view AS
+WITH views AS (
+    SELECT mv_contract_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_contract_award_search INNER JOIN awards a ON
+            a.id = mv_contract_award_search.award_id
+    UNION
+    SELECT mv_grant_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_grant_award_search INNER JOIN awards a ON
+            a.id = mv_grant_award_search.award_id
+    UNION
+    SELECT mv_directpayment_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_directpayment_award_search INNER JOIN awards a ON
+         a.id = mv_directpayment_award_search.award_id
+    UNION
+    SELECT mv_idv_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_idv_award_search INNER JOIN awards a ON
+            a.id = mv_idv_award_search.award_id
+    UNION
+    SELECT mv_loan_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_loan_award_search INNER JOIN awards a ON
+            a.id = mv_loan_award_search.award_id
+    UNION
+    SELECT mv_other_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_other_award_search INNER JOIN awards a ON a.id = mv_other_award_search.award_id
+)
+SELECT
+  DISTINCT views.award_id,
```

Comment:
Is this necessary?

Reply:
yes, because of the way that the recipient hash is calculated we can end up with multiple of the same award with different hashs